### PR TITLE
media-sound/ario: bump to version 1.6

### PR DESCRIPTION
**Pull request by evilscientress**
This is a ebuild / version bump for media-sound/ario-1.6. The ebuild is based on the work of @agicquel and the comments in #11321

/cc @mgorny

### CODE REVIEW

File: media-sound/ario/ario-1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+HOMEPAGE="http://ario-player.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}-player/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dbus debug +idle nls taglib zeroconf"
+
+RDEPEND="dev-libs/glib:2
+	dev-libs/libgcrypt:0
```
Comment (juippis) [inline media-sound/ario/ario-1.6.ebuild]: If this gets linked during compile, it needs to be binded with `:=`.

File: media-sound/ario/ario-1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+pkg_postinst() {
+	xdg_icon_cache_update
+}
+
+pkg_postrm() {
+	xdg_icon_cache_update
+}
+
+src_install() {
```
Comment (juippis) [inline media-sound/ario/ario-1.6.ebuild]: Please arrange these ebuild phases to the order they get executed in, ie src_install before pkg_postinst. Rest looks good.

File: media-sound/ario/ario-1.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	zeroconf? ( net-dns/avahi )"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-util/intltool
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )"
+
+DOCS=( AUTHORS )
+
+src_configure() {
+	econf \
```
Comment (juippis) [inline media-sound/ario/ario-1.6.ebuild]: While fixing the two other minor things, you could maybe convert this to an array form, 
```
	local myconf=(
		--stuff
		$(more stuff)
		etc
	)
	econf "${myconf[@]}"
```

It's less error-prone and prettier, but up to you.

**Comment by juippis** [comment]
Also would you be interested in becoming the maintainer for this package? Add yourself to metadata.xml if so.

File: media-sound/ario/ario-1.6.ebuild (hunk 1 above)
Comment (evilscientress) [reply media-sound/ario/ario-1.6.ebuild]: on closer inspection it turned out ario 1.6 no longer depends on libgcrypt

File: media-sound/ario/ario-1.6.ebuild (hunk 3 above)
Comment (evilscientress) [reply media-sound/ario/ario-1.6.ebuild]: fixed

File: media-sound/ario/ario-1.6.ebuild (hunk 2 above)
Comment (evilscientress) [reply media-sound/ario/ario-1.6.ebuild]: fixed

**Comment by evilscientress** [comment]
@juippis I worked in all things you highlighted in the ebuild, thx for pointing them out the way you did.

And yeah I I'd be interested in becoming the maintainer, I've added myself and the proxy-maint team to metadata.xml.

**Comment by evilscientress** [comment]
I just checked and indeed my bugzilla account used an old email address, fixed now.

**Comment by juippis** [comment]
I squashed your commits while merging. Welcome aboard :)

**Comment by juippis** [comment]
60b9f37423792db315fd0f04cbeba801cb16279e

**Comment by evilscientress** [comment]
yay :)

**Comment by agicquel** [comment]
My apologies. I've been very busy these days and I didn't have the time to fix the ebuild. Thank you @evilscientress ;-)

**Pull request closed without merging**